internal/services/containers/kubernetes_fleet_update_strategy_resource.go

```diff
@@ -0,0 +1,307 @@
+package containers
+
+import (
+	"context"
+	"fmt"
+	"time"
+
+	"github.com/hashicorp/go-azure-helpers/lang/pointer"
+	"github.com/hashicorp/go-azure-helpers/lang/response"
+	"github.com/hashicorp/go-azure-helpers/resourcemanager/commonids"
+	"github.com/hashicorp/go-azure-helpers/resourcemanager/commonschema"
+	"github.com/hashicorp/go-azure-sdk/resource-manager/containerservice/2023-10-15/fleetupdatestrategies"
+	"github.com/hashicorp/terraform-provider-azurerm/internal/sdk"
+	"github.com/hashicorp/terraform-provider-azurerm/internal/tf/pluginsdk"
+	"github.com/hashicorp/terraform-provider-azurerm/internal/tf/validation"
+)
+
+var _ sdk.Resource = KubernetesFleetUpdateStrategyResource{}
+var _ sdk.ResourceWithUpdate = KubernetesFleetUpdateStrategyResource{}
+
+type KubernetesFleetUpdateStrategyResource struct{}
+
+func (r KubernetesFleetUpdateStrategyResource) ModelObject() interface{} {
+	return &KubernetesFleetUpdateStrategyResourceSchema{}
+}
+
+type KubernetesFleetUpdateStrategyResourceSchema struct {
+	FleetId  string                                                         `tfschema:"fleet_manager_id"`
```

Comment:
We should spell this out 
```suggestion
	FleetId  string                                                         `tfschema:"kubernetes_fleet_manager_id"`
```